### ISSUE TITLE
Tell users to localize the app

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#473]
At least link from About.

**Comment by misaochan** [comment]
Agreed, this would be good.

**Comment by maskaravivek** [comment]
Do we have a link that can be put in the app?

**Comment by nicolas-raoul** [comment]
I had done translations in the past, and I thought the tool would be on-par with similar translation tools, but now I just had a look as a new user... and found out it is incredibly cumbersome :-/
Maybe we should have gone for Crowdin after all...

The best we can do is probably to link to a tutorial page in the Wiki, in which we need to explain with screenshots how to create an account, choose your language, "Get permission" (why the hell is that needed?), etc.

**Comment by nicolas-raoul** [comment]
This is the best URL I could find:

https://translatewiki.net/w/i.php?title=Special:Translate&group=commons-android-strings&filter=!translated&action=translate

Even though my web browser is in Japanese (with Accept Locale set to Japanese and not logged in), that URL shows up in _English_, which is rather sad for a translation website. Furthermore, it only shows a cryptic error message "_Translations to this language in this group have been disabled. Reason: English is the source language of this page._" Users have to realize there is a "Translate to" selector on the far right of the page. At this point, unsuspecting volunteers can start translating, but can only save after creating an account (and "getting permission" if I understand correctly?), losing their translations in the process.

**Comment by misaochan** [comment]
Yeah, they need to apply for an account, do a translation, and then when the account is created they will lose all the translations they made previously. :/ https://translatewiki.net/wiki/FAQ#How_do_I_register? I don't even have an account on translatewiki, as my application for one was declined previously due to not being able to do a translation.

I'm really not sure why this restriction was made.

**Comment by whym** [comment]
> I don't even have an account on translatewiki, as my application for one was declined previously due to not being able to do a translation.

Does adding or editing documentation messages (.../qqq) count? For example, I have edited  https://translatewiki.net/wiki/Wikimedia:Commons-android-strings-use_previous/qqq which will help translators to understand the role of the particular message.

**Comment by whym** [comment]
As for the url to show, if the app can figure out the right language tag, we can show a customized url. For Japanese ('ja') the url would be https://translatewiki.net/w/i.php?title=Special:Translate&language=ja&group=commons-android-strings

The rest of the problems mentioned by nicolas-raoul remains, though.

**Comment by nicolas-raoul** [comment]
Ah yes, the localized URL is a great idea :-)
This will at least show people the tasks that need to be completed, rather
than an empty page. From there they will probably be able to figure out the
rest.

**Comment by misaochan** [comment]
> Does adding or editing documentation messages (.../qqq) count? For example, I have edited https://translatewiki.net/wiki/Wikimedia:Commons-android-strings-use_previous/qqq which will help translators to understand the role of the particular message.

Oh, I've finally managed to create an account by digging out my very rusty knowledge of a second language, haha. But yeah, the process is still pretty cumbersome IMO.

The localized URL sounds good to me too.

**Comment by misaochan** [comment]
@nicolas-raoul @whym @Nikerabbit 

There is a question from an endorser at https://meta.wikimedia.org/wiki/Grants:Project/Improve_%27Upload_to_Commons%27_Android_App/Renewal asking how he can help with Russian translations.

I looked at https://translatewiki.net/w/i.php?title=Special:Translate&group=commons-android-strings&filter=%21translated&action=translate&language=ru hoping to be able to give him this link, but it says that nothing needs translations. Do I have the wrong link, or are all Russian translations already in place for the app?

**Comment by nicolas-raoul** [comment]
Yes, all Russian translations are done, but only 49% are reviewed.

**Comment by tanvidadu** [comment]
Is this issue for grabs. I will like to work on it
